routing/CMakeLists.txt

```diff
@@ -185,6 +185,7 @@ set(SRC
 omim_add_library(${PROJECT_NAME} ${SRC})
 
 target_link_libraries(${PROJECT_NAME}
+  base
```

Comment:
Interesting, base library should come as a dependency from other referenced libraries below. Why it doesn't work?

Reply:
Probably related to the fact that some `target_link_libraries` calls in this project don't have PUBLIC/PRIVATE keywords.

Reply:
Then it should be fixed!